src/diffusers/schedulers/scheduling_dpmsolver_singlestep.py

```diff
@@ -239,7 +241,7 @@ def get_order_list(self, num_inference_steps: int) -> List[int]:
             elif order == 1:
                 orders = [1] * steps
         else:
-            if order == 3:
+            if order >= 3:
```

Comment:
why is this change needed here?
if order > 3, we will already throw an error so it is not possible to have `order > 3` here, no?